src/commands/build/wranglerjs/mod.rs

```diff
@@ -54,10 +53,10 @@ pub fn run_build(project: &Project) -> Result<(), failure::Error> {
             wranglerjs_output.project_size()
         );
 
+        fs::remove_file(temp_file)?;
         message::success(&msg);
         Ok(())
     } else {
-        fs::remove_file(temp_file)?;
```

Comment:
if we are preserving this for forensic purposes, should we output its location?

Reply:
I assume that "forensic purposes" means post-moderm analysis, in that case yes. The location is available when using `RUST_LOG=info`. I would prefer that the user don't know about it.